README.md

```diff
@@ -1,51 +1,56 @@
 git clone https://github.com/FlowCrypt/flowcrypt-ios.git && cd flowcrypt-ios
-# this is to setup husky and eslint
+# 2. To set up Husky and ESLint
 npm install
-# set up environment
+# 3. Set up your environment
 sudo xcode-select --install
 curl -sSL https://get.rvm.io | bash -s stable
 rvm install 3.0.2
 rvm --default use 3.0.2
-# install swiftformat
+# 4. Install SwiftFormat
 brew install swiftformat
-# install dependencies
+# 5. Install the dependencies
 make dependencies
 ```
 
-## Development
+## 🖥️ Development
+
+We recommended developing and running tests using only the `Debug FlowCrypt` scheme.
 
-Recommended to develop and run tests only using `Debug FlowCrypt` scheme
+## 🖌️ Code Design
 
-## Code Design
-See [Code Design README](./code-design.md)
+Please refer to our [Code Design README](./code-design.md) to learn how we handle errors and tasks.
 
-## UI Tests
+## 🔍 UI Tests
 
-See [Appium README](./appium/README.md)
+Please refer to our [Appium README](./appium/README.md) for UI tests. It explains how to configure your environment, build the app, and write, run, or debug tests.
 
-## Dependency docs
+## 📚 Dependency docs
 
-- UI - [Texture documentation](https://texturegroup.org/docs/getting-started.html)
-- Storage - [Realm](https://github.com/realm)
-- IMAP/SMTP provider - [MailCore](https://github.com/MailCore/mailcore2)
-- icons - use https://developer.apple.com/sf-symbols/
+- **UI**: We use [Texture](https://texturegroup.org/docs/getting-started.html) to improve the performance, memory efficiency, and thread safety of the app.
+- **Storage**: We use [Realm](https://www.mongodb.com/docs/realm/sdk/swift/realm-database/) as an alternative to SQLite.
+- **IMAP/SMTP provider**: We use [MailCore](http://libmailcore.com/api/objc/index.html) which provides an asynchronous API to work with the e-mail protocols.
+- **Icons**: We use [SF Symbols](https://developer.apple.com/sf-symbols/), a powerful resource for a vast collection of vector icons.
 
-## Admin - generating .ipa for penetration tester
+## 🛠 Generating .ipa for penetration tester (Admin)
 
-1) get uuid of their device and input it into https://developer.apple.com/account/ -> Devices
-2) choose the right build (eg `FlowCrypt Enterprise`) and choose `Any iOS Device (arm64)`
-3) Xcode -> Product -> Archive
-4) Distribute app -> Add Hoc -> Next -> Next (automatically manage signing)
-5) This creates a folder at the target where you export it to, and the IPA will be there
+1\. Get the UUID of their device and input it into the [Apple Developer](https://developer.apple.com/account/) account &#10140; **Devices**.
+2\. Choose the right build (e.g. **FlowCrypt Enterprise**) and choose **Any iOS Device (arm64)**.
+3\. **Xcode** &#10140; **Product** &#10140; **Archive**.
+4\. **Distribute app** &#10140; **Add Hoc** &#10140; **Next** &#10140; **Next** (automatically manage signing).
+5\. This creates a folder at the target where you export it, and the IPA will be there.
```

Comment:
It's currently rendered as a single line

<img width="800" alt="Screenshot 2023-06-15 at 11 26 57" src="https://github.com/FlowCrypt/flowcrypt-ios/assets/1062093/58297a4a-e254-4d35-a171-7e47a4f9b98c">

Reply:
\+